File: Java/toastexample/app/src/main/java/com/example/toast_example/MainActivity.java
package com.example.toast_example;

import androidx.appcompat.app.AppCompatActivity;

import android.app.AlertDialog;
import android.content.DialogInterface;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button toastButton;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        toastButton = (Button) findViewById(R.id.button);
        toastButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(),"Toast now showing", Toast.LENGTH_SHORT).show();
            }
        });

    }
}

File: Kotlin/toastexample/app/src/main/java/com/example/toast_example/MainActivity.kt
package com.example.toast_example

import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var toastButton: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        toastButton = findViewById<View>(R.id.button) as Button
        toastButton!!.setOnClickListener {
            Toast.makeText(
                applicationContext,
                "Toast now showing",
                Toast.LENGTH_SHORT
            ).show()
        }
    }
}